client/verta/verta/_deployment/endpoint.py

```diff
@@ -7,6 +7,8 @@
 import yaml
 from functools import reduce
 
+from verta.external import six
```

Comment:
Relative import is _strongly_ preferred for this project.
```suggestion
from ..external import six
```